src/error.js

```diff
@@ -489,6 +489,11 @@ export function getErrorReportData(message, filename, line, col, error,
   }
   data['rt'] = runtime;
 
+  // Add our a4a id if we are inabox
+  if (runtime === 'inabox') {
+    data['adid'] = getMode().a4aId;
```

Comment:
need to add this field to https://github.com/ampproject/amphtml/blob/master/src/mode.js#L32
fixing the single pass code to actually throw the error

Reply:
in saying that, `runtime` also really needs to be there. the reason why it doesnt cause an error is because we do a `getMode().runtime` conditional before reading a few lines above this

Reply:
Ahhh makes sense. Thank you so much for tracking this down, I'll go ahead and fix this now 👍

Reply:
Done.